avocado/core/sysinfo.py

```diff
@@ -409,6 +409,16 @@ def __init__(self, basedir=None, log_packages=None, profiler=None):
             log.debug('File %s does not exist.', commands_file)
             self.commands = []
 
+        fail_commands_file = config.get('sysinfo.collectibles.fail_commands')
+        if os.path.isfile(fail_commands_file):
+            log.info('Commands configured by file: %s', fail_commands_file)
+            self.fail_commands = genio.read_all_lines(fail_commands_file)
+            self.fail_commands = [
+                cmd for cmd in self.fail_commands if cmd not in self.commands]
+        else:
+            log.debug('File %s does not exist.', fail_commands_file)
```

Comment:
Not sure if debug or error.

Reply:
We had debug for files and commands, so following that with fail_files and fail_commands as well